tests/analyses/test_data.py

```diff
@@ -0,0 +1,59 @@
+from virtool_core.models.enums import QuickAnalyzeWorkflow
+
+from virtool.data.utils import get_data_from_app
+from tests.users.test_data import users_data
+
+
+async def test_create(app, mongo, snapshot, static_time, users_data):
+    """
+    Tests that an analysis is created with the expected fields.
+    """
+    subtractions = ["subtraction_1", "subtraction_2"]
+
+    await users_data.create(password="hello_world", handle="bill")
+    user_info = await mongo.users.find_one({"handle": "bill"})
```

Comment:
A variable name ending with non-specifics like `info`, `data`, etc is a variable naming smell.

---

The full user object is returned when you create user with the data layer or `fake2`. No need to do another query directly to Mongo.

```py
user = await fake2.users.create()

user_id = user.id
```